fe/fe-core/pom.xml

```diff
@@ -576,6 +581,12 @@ under the License.
             </exclusions>
         </dependency>
 
+        <!--  https://mvnrepository.com/artifact/org.apache.spark/spark-catalyst_2.12 -->
```

Comment:
what are these spark related pom changes for?

Reply:
The modification of the spark-related pom is necessary because the jprotobuf-pre-compile plugin scans the imported classes in the project during dynamic generation. This modification helps resolve the ClassNotFoundException error that occurs during compilation.